pkg/kn/commands/broker/describe.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	v1beta1 "knative.dev/eventing/pkg/apis/eventing/v1beta1"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/printers"
+)
+
+var describeExample = `
+  # Describe broker 'mybroker' in the current namespace
+  kn broker describe mybroker
+  # Describe broker 'mybroker' in the 'myproject' namespace
```

Comment:
```suggestion
  kn broker describe mybroker
  
  # Describe broker 'mybroker' in the 'myproject' namespace
```

Reply:
separate the example with a newline

Reply:
Fixed the spacing in all broker cmds, https://github.com/knative/client/pull/894/commits/6f2d98c5dfa983af2c1e1304dc00197ee1b15b50.